src/app/api/services/workflow/workflow.service.ts

```diff
@@ -31,15 +31,16 @@ export class WorkflowService {
     /**
      * Returns the wokflow or workflow actions for a page asset
      *
-     * @param {string} pageIdentifier
+     * @param {string} inode
      * @returns {Observable<Workflow[]>}
      * @memberof WorkflowService
      */
-    getPageWorkflows(_pageIdentifier: string): Observable<Workflow[]> {
-        return Observable.of([
-            { name: 'Workflow 1', id: 'one' },
-            { name: 'Workflow 2', id: 'two' },
-            { name: 'Workflow 3', id: 'three' }
-        ]);
+    getPageWorkflows(inode: string): Observable<DotWorkflowActions[]> {
```

Comment:
Now ```getPageWorkflows``` doesn't makes sense because we're returning ```DotWorkflowActions``` also, WorkflowActions are not tied to page but to any type of content so we can rename this to ```getWorkflowActions```

Reply:
getContentWorkflowActions better

Reply:
Do we have WorkflowActions in any other thing than contents?